src/bot/commands/quotes.ts

```diff
@@ -1,10 +1,14 @@
 import Discord from 'discord.js';
 import MongoService from 'src/core/services/mongo.service';
+import ConfigService from 'src/core/services/config.service';
+import container from 'src/inversity.config';
+import shortid from 'shortid';
```

Comment:
Again for consistency can you keep imports of packages at the top above any local files?